tests/acceptance/features/coreApiSharePublicLink2/uploadToPublicLinkShare.feature

```diff
@@ -1,4 +1,4 @@
-@issue-1276
+@issue-1276 @skipOnReva
```

Comment:
You can remove `@skipOnRevaMaster` and then merge the example tables and `@skipOnReva` tag from scenarios.